src/__tests__/convert-spec.js

```diff
@@ -0,0 +1,62 @@
+import {convertContentTo, convertContentFrom} from '../lib/convert';
+import DraftJS, {ContentState} from 'draft-js';
+
+describe('convertContentTo', () => {
+
+	let draftJsContent;
+	beforeEach(() => {
+		draftJsContent = DraftJS.ContentState.createFromText('<div>some <a href="#">content</a></div>');
+	});
+
+	it('converts DraftJS content to json', () => {
+		expect(draftJsContent).toBeInstanceOf(ContentState);
+		let content = convertContentTo(draftJsContent, 'json');
+		expect(content).toBeInstanceOf(Object);
+	});
+
+	it('converts DraftJS content to html string', () => {
+		expect(draftJsContent).toBeInstanceOf(ContentState);
+		let content = convertContentTo(draftJsContent, 'html');
+		expect(typeof content).toBe('string');
+	});
+
+	it('strips html from a string', () => {
+		let testString = 'TEST_CONTENT';
+		let htmlString = '<a href="#">' + testString + '</a>';
+		let result = convertContentTo(htmlString, 'text');
+		expect(result).toBe(testString);
+	});
+});
+
+describe('convertContentFrom', () => {
+	let draftJsContent;
+	beforeEach(() => {
+		draftJsContent = DraftJS.ContentState.createFromText('<div>some <a href="#">content</a></div>');
```

Comment:
Same as the other comment. if you wanted to work with HTML type, createFromText function won't be the best solution.